modules/ztp-upgrading-gitops-ztp.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: CONCEPT
+[id="ztp-upgrading-gitops-ztp_{context}"]
+= Upgrading GitOps ZTP
+
+You should upgrade GitOps ZTP to the latest version.
```

Comment:
@imiller0, @serngawy We should probably make this a procedure rather than linking to the repo. WDYT?

Reply:
Also note - not allowed link to repos from docs by default: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites

Any repos we want to link to would need explicit approval from Eric Rich.

Reply:
Yes - aware of that. Will contact him.